server/lib/get-ad-targeting.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const fetch = require('./wrap-fetch.js')(require('node-fetch'), {
+	tag: 'getAdTargeting',
+});
+
+module.exports = (uuid) => fetch(`https://${process.env.ADS_API_HOST}/v1/content/${uuid}`, {
+	timeout: 2000,
+})
+	.then(response => response.json())
+	.then(response => {
+		const data = {};
+		data.slot = `/5887/ft.com/${response.dfpSite}`;
```

Comment:
@adgad `dfpSite` and `dfpZone` aren't in the current response format from ads-api, but I'm not sure what to replace them with?